### DISCUSSION TITLE
Alternatives to wxtoimg

### DISCUSSION THREAD

**Discussion by Cadair** [jekhokie/raspberry-noaa-v2#364] [General]
I know there are few things which really have the feature set of wxtoimg, but given it's basically abandonware and not open source how would people feel about making its use optional and adding support for other decoders?

My personal favourite is https://github.com/martinber/noaa-apt although I know there are tons of others out there.

**Comment by dom-robinson** [comment]
Ultimately a good idea - +1 from me :) 

And noaa-apt does seem active and a good code base to start from, although i have not tried it yet myself.

**Comment by dom-robinson** [comment]
https://github.com/jekhokie/raspberry-noaa-v2/pull/440

My next PR in the queue to get finished :)
